pages/learn/getting-started-step-by-step.md

```diff
@@ -490,21 +531,25 @@ The following schema validates a geographical location:
 To reference this schema in the product catalog schema:
 
 1. Inside the `properties` object, add a key named `warehouseLocation`:
-    
+
 ```json
-"properties": {
-...
-  "warehouseLocation": {
+{
+  "properties": {
+  ...
+    "warehouseLocation": {
+    }
   }
 }
```

Comment:
Similarly, this isn't the full schema on its own.  The current snippet (although not proper JSON) demonstrates well that this is an excerpt from the above schema, but the new snippet is a schema by itself.

This is seen in a lot of these changes.